website/content/en/highlights/2023-11-07-new-linux-repos.md

```diff
@@ -0,0 +1,124 @@
+---
+date: "2023-11-07"
+title: "A new home for Linux packages"
+description: ""
+authors: ["spencergilbert"]
+pr_numbers: []
+release: "0.34.0"
+hide_on_release_notes: false
+badges:
+  type: "announcement"
+---
+
+Starting with Vector's `0.34.0` release, the `deb` and `rpm` packages will have
+a new home at `vector.dev` courtesy of Datadog. We'll continue to publish
+packages to both the new and existing locations through the end of **2023**.
+
+{{< warning >}}
+Starting in **2024** we'll stop using `timber.io` and exclusively publish packages
+to `vector.dev`.
+{{< /warning >}}
+
+We aim to make this a seamless transition by providing plenty of time to switch
+repositories, as well as publishing previous versions of Vector to have the new
+repository be a drop in replacement - without updating your Vector version.
+
+If you have any questions or concerns don't hesitate to reach out on [Discord]
+or [Discussions]!
+
+## Migration guide
+
+The following command will **remove** the existing repository and configure the
+new repository.
+
+```sh
+CSM_MIGRATE=true bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script_vector0.sh)"
+```
+
+Alternatively, `CSM_MIGRATE` may be left unset to leave the removal of the
+existing repository to your discretion.
+
+### Manual step by step instructions
+
+<details>
+  <summary>APT</summary>
+1. Remove the existing repository:
+
+```sh
+rm "/etc/apt/sources.list.d/timber-vector.list"
+```
+
+2. Run the following commands to set up APT to download through HTTPS:
+
+```sh
+sudo apt-get update
+sudo apt-get install apt-transport-https curl gnupg
+```
+
+3. Run the following commands to set up the Vector `deb` repo on your system
+and create a Datadog archive keyring:
+
+```sh
+echo "deb [signed-by=/usr/share/keyrings/datadog-archive-keyring.gpg] https://apt.vector.dev/ stable vector-0" | sudo tee "/etc/apt/sources.list.d/vector.list"
+sudo touch /usr/share/keyrings/datadog-archive-keyring.gpg
+sudo chmod a+r /usr/share/keyrings/datadog-archive-keyring.gpg
+curl https://keys.datadoghq.com/DATADOG_APT_KEY_CURRENT.public | sudo gpg --no-default-keyring --keyring /usr/share/keyrings/datadog-archive-keyring.gpg --import --batch
+curl https://keys.datadoghq.com/DATADOG_APT_KEY_F14F620E.public | sudo gpg --no-default-keyring --keyring /usr/share/keyrings/datadog-archive-keyring.gpg --import --batch
+curl https://keys.datadoghq.com/DATADOG_APT_KEY_C0962C7D.public | sudo gpg --no-default-keyring --keyring /usr/share/keyrings/datadog-archive-keyring.gpg --import --batch
+```
+
+4. Run the following commands to update your local `apt` repo and install Vector:
+
+```sh
+sudo apt-get update
+sudo apt-get install vector datadog-signing-keys
```

Comment:
I think right now we're not distributing the datadog-signing-keys package through the vector repos, so let's remove it until we do. We'll have to make some minor modification in it to support vector.

Reply:
Addressed in 9b67ba9c6